pkgs/development/python-modules/django-redis/default.nix

```diff
@@ -66,6 +67,11 @@ buildPythonPackage {
     pytestCheckHook
   ];
 
+  pytestFlagsArray = lib.optionals (pythonAtLeast "3.11") [
+    # DeprecationWarning: 'cgi' is deprecated and slated for removal in Python 3.13
+    "-W" "ignore::DeprecationWarning"
+  ];
```

Comment:
Should we maybe just add that by default to pytestCheckHook? I think it falls in a similar category as Werror which is most useful for upstream developers